components/centraldashboard/public/components/dashboard-view.css

```diff
@@ -81,6 +88,40 @@ paper-card.thin {
     padding: .5em 1em;
 }
 
+paper-icon-item {
+    --paper-item-icon-width: 40px;
+    --paper-item-focused-before: {
+        background: #007dfc;
+    };
+    --paper-icon-button-ink-color: var(--accent-color);
+}
+
+.link {
+    display: block;
+    margin: .5em 1em;
+}
+
+.link paper-icon-item {
+    @apply --layout-flex;
+    @apply --layout-justified;
+    @apply --layout-horizontal-reverse;
+    border: 1px solid var(--border-color);
+    padding: .5em 1em;
+    border-radius: 5px;
+    overflow:hidden;
+    --paper-item-icon: {
+        color: var(--accent-color);
+        background: rgba(0, 125, 252, 0.25);
```

Comment:
The mocks remove the background from the icon button.

Reply:
Fixed and updated